src/wp2hugo/internal/hugogenerator/hugo_custom_css.go

```diff
@@ -0,0 +1,40 @@
+package hugogenerator
+
+import "path/filepath"
+
+const _customCSS = `
+.gallery {
+	display: flex;
+	flex-wrap: wrap;
+}
+.gallery figure,
+.gallery figure img {
+	text-align: center;
+}
+.gallery figure img {
+	margin: 1rem auto;
+}
+.gallery-cols-1 figure {
+	width: 100%;
+}
+.gallery-cols-2 figure {
+	width: 50%;
+}
+.gallery-cols-3 figure {
+	width: 33.3333333333%;
+}
+.gallery-cols-4 figure {
+	width: 25%;
+}
+.gallery-cols-5 figure {
+	width: 25%;
+}
+.gallery-cols-6 figure {
+	width: 16.666666666%;
+}
+`
+
+func setupCSS(siteDir string) error {
+	err := appendFile(filepath.Join(siteDir, _outputCssFile), _customCSS)
```

Comment:
Can we write this into `extend_head.html` to collect all CSS at one place?
https://github.com/ashishb/wp2hugo/blob/24940a53923b882c338d0c72b7ec14b1f0c93e9f/src/wp2hugo/internal/hugogenerator/hugo_custom_font.go#L17

Reply:
I was under the impression that `hugo_custom_font.go` was called conditionaly , depending on arguments, while this needs to run unconditionaly. Was I wrong ?

Reply:
Well, I was wrong. Fixed.